src/compiler/emitter.ts

```diff
@@ -7093,9 +7097,11 @@ const _super = (function (geti, seti) {
 
                     write(text);
                 }
-                write(`], function(${exportFunctionForFile}) {`);
+                write(`], function(${exportFunctionForFile}, ${contextObjectForFile}) {`);
                 writeLine();
                 increaseIndent();
+                write(`var __moduleName = ${contextObjectForFile}.id;`);

```

Comment:
this should be `var __moduleName = ${contextObjectForFile} && ${contextObjectForFile}.id;` to avoid breaking users of older JSPM versions.